Automattic/jetpack
# CLI: avoid PHP notices when running connection tests

#### Changes proposed in this Pull Request:

* Some tests do not have a message, so we should take that into account before we try to display them.

#### Other information:


#### Jetpack product discussion

* N/A

#### Does this pull request change what data or activity we track or use?

* No

#### Testing instructions:

* On a site that's connected to WordPress.com, run `wp jetpack status`
* Ensure that there are no notices in your logs.
